lib/index.js

```diff
@@ -0,0 +1,261 @@
+'use strict';
+
+const path = require('path');
+const querystring = require('querystring');
+const fetch = require('node-fetch');
```

Comment:
I see what you mean - so give the user an option to pass in a model, but default to something like https://github.com/UKHomeOffice/end-tenancy/blob/master/apps/end-tenancy/models/postcodes.js? How would you deal with things like the auth code we currently set as a secret? Also the end tenancy implementation has a requirement that the address is in England, which involves a further api call.